src/app/api/v1/fxa-rp-events/route.ts

```diff
@@ -228,6 +237,29 @@ export async function POST(request: NextRequest) {
           event,
           updatedSubscriptionFromEvent,
         });
+
+        // get profile id
+        const result = await getOnerepProfileId(subscriber);
+        const oneRepProfileId = result[0]["onerep_profile_id"] as number;
```

Comment:
Q: Do we know if `result` array will have at least one item here?
```js
async function getOnerepProfileId (subscriberId) {
  return await knex('subscribers')
    .select('onerep_profile_id')
    .where('id', subscriberId)
}
```

Reply:
added optional chaining, thanks